.env.local.example

```
@@ -44,6 +44,8 @@ NEXT_PUBLIC_ARBITRUM_RPC=
 NEXT_PUBLIC_ETHERLINK_RPC=
 NEXT_PUBLIC_HYPEREVM_RPC=
 NEXT_PUBLIC_MONAD_RPC=
+NEXT_PUBLIC_KATANA_RPC=
+NEXT_PUBLIC_KATANA_RPC_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_KATANA_RPC_API_KEY key should go before the NEXT_PUBLIC_MONAD_RPC key

(UnorderedKey)